### ISSUE TITLE
Dear friend, could not you help me with the problem

### ISSUE THREAD

**Issue by xxxvik-xakerxxx** [addy-dclxvi/void-bspwm-dotfiles#1]
I  use a freshly installed void, and i heve problems with displaying icons in the polybar, in the screenshot you configuration, while in the system icons are displayed

![2018-04-26-121240_1024x600_scrot](https://user-images.githubusercontent.com/19531907/39297196-8c843b30-494b-11e8-8f89-1ad7236fc178.png)

**Comment by addy-dclxvi** [comment]
Ohh Yes. In that configuration I was still using font Awesome 4. The current version of Font Awesome is version 5. All the character glyphs from Font Awesome 4 are depreciated.

And my current setup of Bspwm has been changed a lot. I use Lemonbar right now. Without any character glyphs.

Sorry for making You disappointed 😔

**Comment by xxxvik-xakerxxx** [comment]
thanks friend, but even with awesome 4 does not work, and in other configurations too

**Comment by addy-dclxvi** [comment]
Font Awesome 4 also has several version 🤔

**Comment by xxxvik-xakerxxx** [comment]
yes, but why are the symbols in the mousepad visible and not in the panel?

**Comment by xxxvik-xakerxxx** [comment]
could you tell me what kind of bar in this photo
https://raw.githubusercontent.com/addy-dclxvi/almighty-dotfiles/master/preview-toys.jpg

**Comment by addy-dclxvi** [comment]
How could it be possible 😱

Please check the Xft name of your current installed Font Awesome. Maybe it's different with the Xft name I set on Polybar config. 🤔

If it's different, change the Xft font name in the Polybar config.

**Comment by addy-dclxvi** [comment]
It's tint2. I name it Liness in my tint2-themes-collection repo.

**Comment by xxxvik-xakerxxx** [comment]
> Please check the Xft name of your current installed Font Awesome. Maybe it's different with the Xft name I set on Polybar 

> 

what do you mean?
![2018-04-26-131612_1024x600_scrot](https://user-images.githubusercontent.com/19531907/39300397-2f731d5e-4954-11e8-999b-28fd654ae400.png)

**Comment by addy-dclxvi** [comment]
`fc-list -v`

Then search for fontawesome. The command above list the Xft name of the fonts.

**Comment by xxxvik-xakerxxx** [comment]
![2018-04-27-075308_1024x600_scrot](https://user-images.githubusercontent.com/19531907/39345290-51b01bb4-49f0-11e8-9755-57a0f14accd5.png)

that's all there is

**Comment by addy-dclxvi** [comment]
It seems You're using FontAwesome 5 🤔

Try to change `font-1 = FontAwesome:pixelsize=8` in Polybar config with `font-1 = FontAwesome 5 Free:pixelsize=8`

I can't test it because I'm not using Polybar & FontAwesome anymore.

**Comment by xxxvik-xakerxxx** [comment]
look at the dear friend there is FontAwesome.ttf not 5 versions, okay I still refused so far from the polybar
![2018-04-27-084858_1024x600_scrot](https://user-images.githubusercontent.com/19531907/39346686-19641866-49f8-11e8-8464-4c81141c03b3.png)
thanks for the help)
